templates/kubernetes/docs/1.20/release-notes.md

```diff
@@ -13,6 +13,33 @@ layout: [base, ubuntu-com]
 toc: False
 ---
 
+# 1.20+ck1 Bugfix release
+
+### February 23rd, 2021 - [charmed-kubernetes-596](https://api.jujucharms.com/charmstore/v5/charmed-kubernetes-596/archive/bundle.yaml)
+
+## Fixes
+
+A list of bug fixes and other minor feature updates in this release can be found at
+[https://launchpad.net/charmed-kubernetes/+milestone/1.20+ck1](https://launchpad.net/charmed-kubernetes/+milestone/1.20+ck1).
+
+## Notes / Known Issues
+
+- Secret names
```

Comment:
Might be too much of a pain but if you marked these down with definition lists it looks a little better:
![image](https://user-images.githubusercontent.com/1413534/112029202-e618ad00-8b30-11eb-910e-10a6249b71d8.png)

If you want the markup:
```html
<dl>
  <dt>Secret names</dt>
  <dd><p><a href="https://bugs.launchpad.net/charm-kubernetes-master/+bug/1906732">LP 1906732</a>
highlighted an issue where <code>kubernetes-worker</code> units would overwrite existing
secrets when deployed as different application names. This lead to some worker
units losing the ability to authenticate with the cluster. This has be resolved
by ensuring new secrets are uniquely named in the form: <code>auth-$username-$random</code>.</p></dd>
</dl>
<dl>
<dt>Juju and GCP</dt>
<dd><p><a href="https://bugs.launchpad.net/juju/+bug/1761838">LP 1761838</a> describes an issue
with Juju and Google cloud where deployments may fail due to FAN networking.
Workaround this by disabling FAN configuration for Google cloud models:</p>
  <p><code>juju model-config -m &lt;model_name&gt; fan-config="" container-networking-method=""</code></p>
</dd>
</dl>
```